litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowRunDetailsTable/TableToolbar.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Typography,
+  InputBase,
+  InputAdornment,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Button,
+  IconButton,
+} from '@material-ui/core';
+import React, { ChangeEvent, useRef, useState } from 'react';
+import SearchIcon from '@material-ui/icons/Search';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import useStyles from './styles';
+import DateRangeSelector from '../../../../components/DateRangeSelector';
+
+interface TestCallBackType {
+  (clusterName: string): void;
+}
+
+interface TestResultCallBackType {
+  (testResult: string): void;
```

Comment:
improve this callback situation wherever it is used

Reply:
Actually its more a state passing function than callback, have named it a callback as it functions similarly.. @arkajyotiMukherjee